src/web/mjs/connectors/Alphapolis.mjs

```diff
@@ -63,4 +67,18 @@ export default class Alphapolis extends Connector {
             };
         });
     }
+
+    async _handleConnectorURI(payload) {
+        let request = new Request(payload.hiresPicture, this.requestOptions);
```

Comment:
Should test if highres url is not null/empty before sending a request

Reply:
Its always filled, its the default value.